state/metadata/state_manager_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package metadata_test
```

Comment:
nit: update to `metadata` should only use `_test` suffix if needed to avoid a cyclic dependency

Reply:
Done